src/dashboard/apps/consent/tests/test_models.py

```diff
@@ -1,63 +1,97 @@
 """Dashboard consent models tests."""
 
-from datetime import timedelta
+import datetime
 
 import pytest
-from django.utils import formats
+from django.db.models import signals
+from django.utils.timezone import make_aware
 
-from apps.auth.factories import UserFactory
 from apps.consent import AWAITING, REVOKED, VALIDATED
 from apps.consent.factories import ConsentFactory
+from apps.consent.models import Consent
+from apps.consent.signals import handle_new_delivery_point
+from apps.consent.utils import consent_end_date
 from apps.core.factories import DeliveryPointFactory
+from apps.core.models import DeliveryPoint
 
 
 @pytest.mark.django_db
 def test_create_consent():
     """Tests the creation of a consent."""
```

Comment:
In fact you are testing the consent creation using a delivery point signal :wink: